### DISCUSSION TITLE
Tool to convert OPENVSP data to jsbsim

### DISCUSSION THREAD

**Discussion by www2000** [JSBSim-Team/jsbsim#605] [Ideas]
Hi i like to see a tool to create jsb FDM (xml) files from data that is generate by OpenVSP.

**Comment by seanmcleod** [comment]
Take a look at http://www.chateau-logic.com/content/using-vspaero-generate-aerodynamic-model-jsbsim-flightgear

**Comment by www2000** [comment]
Sorry for the late responds.
Thanks i know this article.
I think this tool need be include in the rewrite of jsbsimcommander.
